lighthouse-cli/test/fixtures/dobetterweb/dbw_tester.js

```diff
@@ -21,22 +21,22 @@
 
 const params = new URLSearchParams(location.search);
 
-if (params.has('dateNow')) {
+if (location.search === '' || params.has('dateNow')) {
```

Comment:
gotcha

another option could be `params.toString() === ''` which i slightly prefer, but not hugely.